### PR TITLE
Hall of fame

### PR DESCRIPTION
**Pull request by ThomasKranitsas**

### CODE REVIEW

**Comment by ThomasKranitsas** [comment]
@birdofpreyru can you please approve this one?

File: src/shared/components/tco/HallOfFamePage/Champions/developmentAndFirst2finish.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
+$primary: #5cc900;
+$champions: #effae4;
+$empty: #fafdf6;
```
Comment (birdofpreyru) [inline src/shared/components/tco/HallOfFamePage/Champions/developmentAndFirst2finish.scss]: $primary > $tc-green
$champions > $tc-green-10
$empty > ???

File: src/shared/components/tco/HallOfFamePage/Finalists/algorithmAndMarathon.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
+$winner: #fd7d00;
+$border: #ffe3cf;
+$finalists: #fff0eb;
+$empty: #fffaf9;
```
Comment (birdofpreyru) [inline src/shared/components/tco/HallOfFamePage/Finalists/algorithmAndMarathon.scss]: $winner > $tc-orange
$border > $tc-orange-30
$finalists > $tc-orange-10
$empty > ???

File: src/shared/components/tco/HallOfFamePage/EventCarousel/index.jsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
+/**
+ * Carousel that shows the logos of all the TCO events and allows a user to click
+ * through and select one.  Mobile mode renders a scrollable div with all events.
+ */
+import _ from 'lodash';
```
Comment (birdofpreyru) [inline src/shared/components/tco/HallOfFamePage/EventCarousel/index.jsx]: We definitely need to create a generic Carousel component for our code.

File: src/shared/components/tco/HallOfFamePage/Finalists/developmentAndFirst2finish.scss [hunk 4]
```diff
@@ -0,0 +1,22 @@
+$winner: #5cc900;
+$border: #cef0af;
+$finalists: #effae4;
+$empty: #fafdf6;
```
Comment (birdofpreyru) [inline src/shared/components/tco/HallOfFamePage/Finalists/developmentAndFirst2finish.scss]: Colors!

File: src/shared/components/tco/HallOfFamePage/Finalists/uiDesignAndPrototype.scss [hunk 5]
```diff
@@ -0,0 +1,22 @@
+$winner: #15acec;
+$border: #bce6f9;
+$finalists: #e9f6fc;
+$empty: #f8fcfe;
```
Comment (birdofpreyru) [inline src/shared/components/tco/HallOfFamePage/Finalists/uiDesignAndPrototype.scss]: Colors!

File: src/shared/components/tco/HallOfFamePage/TripWinners/allRoles.scss [hunk 6]
```diff
@@ -0,0 +1,15 @@
+$role: #0681ff;
+$winners: #e2f0ff;
+$empty: #f8fbff;
```
Comment (birdofpreyru) [inline src/shared/components/tco/HallOfFamePage/TripWinners/allRoles.scss]: Colors!

**Pull request merged**